pkg/parser/parser.peg

```diff
@@ -363,6 +363,81 @@ StandaloneAttributes <- attributes:(ElementAttribute)+ BlankLine* EOF { // stand
     return types.NewAttributes(attributes)
 }
 
+// ------------------------------------------
+// Quoted Strings (between curly single or double quotes)
+// ------------------------------------------
+
+QuotedString <- qs:(SingleQuotedString / DoubleQuotedString) {
```

Comment:
you can just write it as 
```suggestion
QuotedString <- SingleQuotedString / DoubleQuotedString
```
ie, without capturing and returning the result in `qs`

Reply:
Done.